packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -284,18 +284,18 @@ class Doctor {
       DoctorResultEvent(validator: validator, result: result).send();
 
       if (result.statusInfo != null) {
-        printStatus('${result.coloredLeadingBox} ${validator.title} (${result.statusInfo})',
+        printStatus('${showColor ? result.coloredLeadingBox : result.leadingBox} ${validator.title} (${result.statusInfo})',
```

Comment:
Avoid printing the terminal color markers in the GitHub template or crash log.

Reply:
Consider using a local variable for `showColor ? result.coloredLeadingBox : result.leadingBox`.